.github/workflows/alert-community.yml

```diff
@@ -1,14 +1,16 @@
 name: "[service] Alert Community"
 
 on:
-  pull_request:
+  pull_request_target:
    types: [opened] 
 
 jobs:
   comment:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          ref: master
```

Comment:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target always runs against the target back which in this repo wont contain malicious code. I think it's fine to leave the default

```suggestion
```